tests/lib/snaps/store/test-snapd-curl/snapcraft.yaml

```diff
@@ -0,0 +1,22 @@
+name: test-snapd-curl
+version: '7.58'
+summary: Curl is a command-line tool for transferring data.
+description: |
+  Curl is a command-line tool for transferring data specified with URL syntax.
+grade: stable
+confinement: strict
+base: core18
+
+apps:
+  curl:
+    command: bin/curl
+    plugs:
+      - network
+      - home
```

Comment:
I think we need snapd-control interface as well? Some tests seems to be failing because of that, e.g.

2021-10-05T21:11:16.1125367Z ++ sudo snap run test-snapd-curl.curl --unix-socket /run/snapd.socket http://localhost/v2/quotas/group-one
2021-10-05T21:11:16.1126551Z curl: (7) Couldn't connect to server